app/frontend/src/ScreenReader/index.js

```diff
@@ -5,12 +5,99 @@ import { ContentContext } from '../lib/contexts'
 
 import './index.css'
 
+const VishraamJsx = word => {
+  switch ( word.slice( -1 ) ) {
+    case ';':
+      return <span className="heavy">{word.slice( 0, -1 )}</span>
+    case ',':
+      return <span className="medium">{word.slice( 0, -1 )}</span>
+    case '.':
+      return <span className="light">{word.slice( 0, -1 )}</span>
+    default:
+      return <>{word}</>
+  }
+}
+
 const ScreenReader = () => {
   const { shabad, bani } = useContext( ContentContext )
 
   const { lines = [] } = shabad || bani || {}
 
-  return lines.map( ( { gurmukhi, id } ) => <p key={id}>{gurmukhi}</p> )
+  const titlesFuzzy = [
+    '<>',
+    '] jpu ]',
+    'pwiqswhI 10',
+    '] qÍ pRswid ]',
+    '] qÍ pRswid kQqy ]',
+    'BujMg pRXwq CMd ]',
+    '] cwcrI CMd ]',
+    '] sv`Xy ]',
+    '] cOpeI ]',
+    'mhlw 1',
+    'mhlw 2',
+    'mhlw 3',
+    'mhlw 4',
+    'mhlw 5',
+    'mhlw 6',
+    'mhlw 7',
+    'mhlw 8',
+    'mhlw 9',
+    'mÚ 1',
+    'mÚ 2',
+    'mÚ 3',
+    'mÚ 4',
+    'mÚ 5',
+    'mÚ 6',
+    'mÚ 7',
+    'mÚ 8',
+    'mÚ 9',
+  ]
+  const titlesExact = [
+    'jwpu ]',
+    'sloku ]',
+    'pauVI ]',
+    'AstpdI ]',
+    'cwcrI CMd ]',
+    'eyk ACrI CMd ]',
+    'sÍYXw ]',
+    'dohrw ]',
+    'sRI BgauqI jI shwie ]',
+    'vwr sRI BgauqI jI kI ]',
+    'rhrwis swihb',
+  ]
+
+  return (
+    <div className="lines-container">
+      <div className="lines">
+        {lines.map( ( { gurmukhi, id } ) => {
+          const isTitle = titlesFuzzy.some( ele => gurmukhi.indexOf( ele ) >= 0 )
```

Comment:
```suggestion
          const isTitle = titlesFuzzy.some( t => gurmukhi.includes( t ) )
```

`.includes` is a more direct way of achieving the same result!